src/Microsoft.Data/BaseColumn.cs

```diff
@@ -93,8 +93,20 @@ public void SetName(string newName, DataFrame dataFrame = null)
 
         public virtual Dictionary<TKey, ICollection<long>> GroupColumnValues<TKey>() => throw new NotImplementedException();
 
+        /// <summary>
+        /// Returns a DataFrame containing counts of unique values
+        /// </summary>
+        public virtual DataFrame ValueCounts() => throw new NotImplementedException();
+
         public virtual GroupBy GroupBy(int columnIndex, DataFrame parent) => throw new NotImplementedException();
 
+        /// <summary>
+        /// Returns a new column with nulls replaced by value
+        /// </summary>
+        /// <remarks>Tries to convert value to the column's DataType</remarks>
+        /// <param name="value"></param>
```

Comment:
Fyi, if you want completeness when it comes to documentation, the triple slash comments should contain all the params being documented, along with a return tag to describe what's being returned.

You also would want to document the exception behavior of the API (at least the ones that have been implemented), for example argumentnullexception/etc.

**If complete documentation is being deferred for later, that is fine.** Just something to remember. When initially implementing/documenting the JSON APIs, not all of the "required" elements (such as `<returns>`) were initially commented, and that lead to us having to go back and add the elements to the docs repo later.

E.g.:  https://github.com/dotnet/dotnet-api-docs/blob/7fff15ca4397fef8fddce8c38cfa1ae3d14a885e/xml/System.Text.Json/JsonElement.xml#L70-L76

cc @carlossanlop